app-misc/goobook/goobook-3.3.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+PYTHON_COMPAT=( python3_6 )
+
+inherit distutils-r1 readme.gentoo-r1
+
+DESCRIPTION="Access your Google contacts from the command line"
+HOMEPAGE="https://gitlab.com/goobook/goobook"
+SRC_URI="mirror://pypi/g/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
```

Comment:
This SLOT is redundant and can be removed. (`SLOT=0` is assumed.)

Reply:
repoman complains:
```
  SLOT.invalid [fatal]          1
   app-misc/goobook/goobook-3.3.ebuild: SLOT: invalid value: ''
```
Also [devmanual](https://devmanual.gentoo.org/general-concepts/slotting/index.html) is explicit:
`Most packages have no need for slotting. These packages specify SLOT="0" in the ebuilds. This is not the same as specifying an empty slot (SLOT="") — an empty slot means "disable slotting entirely", and should not be used. `

Reply:
Huh, I wonder when that changed. I'm getting old and that wasn't a thing when I first started, so never mind.